docker-compose.yml

```diff
@@ -21,18 +21,90 @@ slave:
         - jenkins
     volumes:
         - ./slave:/home/slave
-    volumes_from:
-        - git
     environment:
         - JAVA_HOME=/etc/alternatives/java_sdk
         - SERVICE_NAME=SPACENAME-omero
         - DOCKER_FIX='   '
+    command: -labels centos6 -labels slave -executors 4
 
-nginx:
-    image: nginx
+server:
+    build: server
+    links:
+        - jenkins
+        - pg
+    volumes:
+        - ./server:/home/omero
+    environment:
+        - JENKINS_LABEL=/etc/alternatives/java_sdk
+        - JENKINS_MODE=exclusive
+        - SERVICE_NAME=SPACENAME-server
```

Comment:
As I understood, this container will be the primary server against which the Web will be deployed and against which manual testing will be performed daily. To not break the existing testing workflow, I propose to move `SERVICE_NAME=SPACENAME-omero` to refer to this container and drop the `SERVICE_NAME` from the slave container as the integration server should not need to be accessed via DNS.